sklearn/datasets/_samples_generator.py

```diff
@@ -430,6 +430,17 @@ def make_multilabel_classification(
     p_w_c : ndarray of shape (n_features, n_classes)
         The probability of each feature being drawn given each class.
         Only returned if ``return_distributions=True``.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_multilabel_classification
+    >>> X, y = make_multilabel_classification(random_state=42)
```

Comment:
Hi @rprkh thanks as always for your PR! I guess we can set a custom `n_label=3` to better understand in a glimpse how it affects the elements in `y`.